winutil.ps1

```diff
@@ -1146,7 +1146,7 @@ $WPFundoall.Add_Click({
         Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced\People" -Name "PeopleBand" -Type DWord -Value 1
 
         Write-Host "Changing default Explorer view to Quick Access..."
-        Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced" -Name "LaunchTo" -Type DWord -Value 1
+        Set-ItemProperty -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced" -Name "LaunchTo" -Type DWord -Value 0
```

Comment:
This is based on personal preference, I hate quick access, but I can see others do like it. Bit torn on this change.

Reply:
Isn't this the "restore" portion of the code? The tweak provided is to make the "This PC" the default, but then here in the undo step it also set "This PC" instead of restoring the default settings

Reply:
/facepalm... was just reading the line from my notifications. This is in the undo all section. Thank you for the correction!